src/commands/account/import_account/mod.rs

```diff
@@ -1,42 +1,64 @@
 use inquire::{CustomType, Select};
 use std::{str::FromStr, vec};
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod from_private_key;
+mod from_seed_phrase;
+mod from_web_wallet;
 
 #[derive(Debug, Clone, interactive_clap::InteractiveClap)]
 #[interactive_clap(context = crate::GlobalContext)]
-pub struct Login {
-    #[interactive_clap(named_arg)]
-    ///Select network
-    network_config: crate::network::Network,
+pub struct ImportAccountCommand {
+    #[interactive_clap(subcommand)]
+    import_account_actions: ImportAccountActions,
+}
+
+impl ImportAccountCommand {
+    pub async fn process(&self, config: crate::config::Config) -> crate::CliResult {
+        self.import_account_actions.process(config).await
+    }
+}
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = crate::GlobalContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How would you like to import the account?
+pub enum ImportAccountActions {
+    #[strum_discriminants(strum(
+        message = "import-account-from-web-wallet          - Import existing account (a.k.a. \"sign in\")"
+    ))]
+    /// Import existing account (a.k.a. "sign in")
+    ImportAccountFromWebWallet(self::from_web_wallet::LoginFromWebWallet),
+    #[strum_discriminants(strum(
+        message = "import-account-from-seed-phrase         - Import existing account from seed phrase"
+    ))]
+    /// Import existing account from seed phrase
+    ImportAccountFromSeedPhrase(self::from_seed_phrase::LoginFromSeedPhrase),
+    #[strum_discriminants(strum(
+        message = "import-account-from-private-key         - Import existing account from private key"
+    ))]
+    /// Import existing account from private key
+    ImportAccountFromPrivateKey(self::from_private_key::LoginFromPrivateKey),
```

Comment:
Let's shorten the commands and also use "using" instead of "from" as it reads more naturally this way:

```suggestion
        message = "using-web-wallet          - Import existing account using NEAR Wallet (a.k.a. \"sign in\")"
    ))]
    /// Import existing account using NEAR Wallet (a.k.a. "sign in")
    UsingWebWallet(self::using_web_wallet::LoginFromWebWallet),
    #[strum_discriminants(strum(
        message = "using-seed-phrase         - Import existing account using a seed phrase"
    ))]
    /// Import existing account using a seed phrase
    UsingSeedPhrase(self::using_seed_phrase::LoginFromSeedPhrase),
    #[strum_discriminants(strum(
        message = "using-private-key         - Import existing account using a private key"
    ))]
    /// Import existing account using a private key
    UsingPrivateKey(self::using_private_key::LoginFromPrivateKey),
```